jobs/kpi-forecasting/kpi_forecasting/tests/test_performance_analysis.py

```diff
@@ -0,0 +1,289 @@
+import pytest
```

Comment:
Should we have tests for `write` and/or `fetch`?

Reply:
I'd argue that, in the absence of an easy way to mock querying BQ tables, it'd be more trouble than it's worth.  I did my best to put anything nontrivial that is happening outside of BQ in its own method so I could unit test that to minimize the amount in these functions.